NEWS.md

```diff
@@ -24,6 +24,14 @@ Language changes
     This can be changed back to the original color by setting the environment variable `JULIA_INFO_COLOR` to `"blue"`.
     One way of doing this is by adding `ENV["JULIA_INFO_COLOR"] = :blue` to the `.juliarc.jl` file.
     For more information regarding customizing colors in the REPL, see this [manual section]( http://docs.julialang.org/en/latest/manual/interacting-with-julia/#customizing-colors).
+  * Multiline and singleline command macros have been added. A command macro is
+    like a string macro, but the syntax uses backquotes (<code>`</code>)
```

Comment:
Ugh, markdown makes it really difficult to type backquotes in code. I think I need to escape it like <code>\<code\>\\\`\</code\></code>. Unless anyone has a better solution?